docs/api/access-control.md

```diff
@@ -38,8 +38,8 @@ see the [Access Control Guide](../../guides/access-control.md) or the
 
 There are two ways of specifying Access Control:
 
-1.  List level
-1.  Field level
+1. List level
+2. Field level
```

Comment:
It doesn't like the additional space on list items. One option would be to run the `remark` output through prettier bu running `remark` first